lib/verify.rb

```diff
@@ -0,0 +1,22 @@
+require './lib/order.rb'
+
+class Verify
+
+attr_reader :completed_order
+
+# def initialize(order_klass: Order)
+#   @current_order = []
+#   @current_menu = []
+#   @order_klass = order_klass
+# end
+
+def collate_order(current_order)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `current_order`. If it's necessary, use `_` or `_current_order` as an argument name to indicate that it won't be used. You can also write as `collate_order(*)` if you want the method to accept any arguments but don't care about them. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)